### PR TITLE
[WIP] Fix OPF files reading issue

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**
## Checklist

- [ ] Appropriate tests were added
- [ ] Any code changes were done in a way that does not break public API
- [ ] All documentation related to code changes were updated
- [ ] The new code follows the
  [contributor guidelines](https://github.com/SciML/.github/blob/master/CONTRIBUTING.md), in particular the [SciML Style Guide](https://github.com/SciML/SciMLStyle) and
  [COLPRAC](https://github.com/SciML/COLPRAC).
- [ ] Any new documentation only uses public API

### CODE REVIEW

**Comment by Vaibhavdixit02** [comment]
This looks stuck https://buildkite.com/julialang/scimlbenchmarks-dot-jl/builds/2302#018ef481-1c50-4810-a196-9a51bb631f0d/359-366, should I do a fresh build or do we expect this to not work right now?

**Comment by ChrisRackauckas** [comment]
Received cancellation? Yeah try it again.

**Comment by Vaibhavdixit02** [comment]
It was stuck trying to find the private key for github

**Comment by ChrisRackauckas** [comment]
Timed out but seems like it worked? Just drop the max size a bit?

**Comment by Vaibhavdixit02** [comment]
It was the github key not found error I mentioned above https://buildkite.com/julialang/scimlbenchmarks-dot-jl/builds/2306#018ef7fd-ef94-4ada-85d8-74c6fc55b8f0/359-366

**Comment by ChrisRackauckas** [comment]
@thazhemadam do you know what that could be?

**Comment by Vaibhavdixit02** [comment]
@thazhemadam any leads for how the github key error here can be avoided?

**Comment by ChrisRackauckas** [comment]
What's the current state of this?

**Comment by Vaibhavdixit02** [comment]
Can you move your PowerModels fork to SciML, I think that might solve the GitHub key issue from before

File: benchmarks/OptimizationFrameworks/optimal_powerflow.jmd [hunk 1]
```diff
@@ -717,19 +718,20 @@ function build_opf_mtk_prob(dataset)
 
     optsys = ModelingToolkit.OptimizationSystem(loss, vars, [], constraints=cons, name=:rosetta)
     optsys = ModelingToolkit.structural_simplify(optsys)
-    u0map = Dict([k => 0.0 for k in collect(unknowns(optsys))])
+    u0map = [Num(k) => 0.0 for k in collect(optsys.unknowns)]
+    ks = collect(Num.(optsys.unknowns))
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:722]:
```suggestion
    ks = collect(Num.(unknowns(optsys)))
```

File: benchmarks/OptimizationFrameworks/optimal_powerflow.jmd [hunk 2]
```diff
@@ -717,19 +718,20 @@ function build_opf_mtk_prob(dataset)
 
     optsys = ModelingToolkit.OptimizationSystem(loss, vars, [], constraints=cons, name=:rosetta)
     optsys = ModelingToolkit.structural_simplify(optsys)
-    u0map = Dict([k => 0.0 for k in collect(unknowns(optsys))])
+    u0map = [Num(k) => 0.0 for k in collect(optsys.unknowns)]
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:721]:
```suggestion
    u0map = [Num(k) => 0.0 for k in collect(unknowns(optsys))]
```

Comment (Vaibhavdixit02) [reply benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:721]: I forgot to revert this

**Comment by Vaibhavdixit02** [comment]
Seems to be working now so I'll merge so it can just finish up on master and get updated in the docs output to look at it

**Pull request merged**